File: frontend/src/repository/settings/model/SettingsJson.ts
export type SettingsJson = {
  taskLimit: number;
  sortable: boolean;
  topicColor: boolean;
  mode: 'dark'| 'white';
  topicListType: 'list' | 'icon';
  notificationIntervalSec: number;
  myColor: string;
  stopNotification: boolean;
};

File: backend/infrastructure/application/ginapp/model/SettingsJson.go
package model

type SettingsJson struct {
	TaskLimit               int    `json:"taskLimit"`
	Sortable                bool   `json:"sortable"`
	TopicColor              bool   `json:"topicColor"`
	Mode                    string `json:"mode"`
	TopicListType           string `json:"topicListType"`
	NotificationIntervalSec int    `json:"notificationIntervalSec"`
	MyColor                 string `json:"myColor"`
	StopNotification        bool   `json:"stopNotification"`
}
